builder/test-integration/spec/hypriotos-image/cloud-init_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+describe package('cloud-init') do
+  it { should be_installed }
+end
+
+describe file('/boot/user-data') do
```

Comment:
Should we also add a test for file `/boot/meta-data` even if it's empty?

Reply:
Good idea as it is mandatory to make cloud-init read the user-data file.